app/controllers/AlertController.java

```diff
@@ -193,7 +189,7 @@ public Result query(
         return ok(Json.toJson(new PagedContainer<>(
                 result.values()
                         .stream()
-                        .map(this::internalModelToViewModel)
+                        .map(Alert::toView)
```

Comment:
This seems inverted; internal model should not know about the view or storage models.